DataAccessLibrary/SqlDataAccess.cs

```diff
@@ -14,39 +14,45 @@ namespace DataAccessLibrary
     public class SqlDataAccess : ISqlDataAccess
     {
         private readonly IConfiguration _configuration;
-        public string ConnectionStringName { get; set; } = "Default";
-
+        public string ConnectionStringName { get; } = "SitMeDatabase";
+        public string ConnectionString { get; }
+        public bool ConnectionAvailable { get; }
         public SqlDataAccess(IConfiguration configuration)
         {
             _configuration = configuration;
+            ConnectionString = _configuration.GetConnectionString(ConnectionStringName);
+            try
```

Comment:
Może fajnie by było użyć filtru? 

https://www.infoworld.com/article/2994111/how-to-handle-errors-in-aspnet-web-api.html

W prawdziwym życiu jest tak, że Frontend obsługuje wyjątki z webapi, np wyświetlając userowi "Coś poszło nie tak..." etc. Treść wyjątku nie powinna wyciekać do Clienta. 

Możesz napisać własny filtr który będzie przechwytywał każdy wyjątek w aplikacji. Co możesz wtedy zrobić?

1. Zwracać jeden, stały wyjątek do frontu (np jakiś swój własny, `AppErrorException` jako aggregate exception, w środku error texty z aplikacji które mają sę wyświetlać dla użytkownika)
2. zalogować stacktrace i jakieś dodatkowe informacje.

Jak będziesz dopisywać kiedyś loggera w swoich apkach dobrą praktyką jest jakiś słownik. Np wiadomość w wyjątku zaczynasz od jakiegoś słowa kluczowego:

`SQL - message, stracktrace, datetime`
`NOTIFICATION_SERVICE- message, stakctrace, datetime`

Po takich kluczach łatwiej jest szukać błędów na produkcji :)